simple-query-builder-core/src/main/java/io/github/hbaykuslar/core/Spec.java

```diff
@@ -11,17 +11,17 @@ public class Spec {
 
 
     public Spec join(String joinStatement) {
-        joins.add(joinStatement.trim());
+        joins.add(" join " + joinStatement.trim());
```

Comment:
Your changes are legit solutions for the bug, but I prefer to keep SQL elements in lists without preceding or leading spaces and add necessary space characters in the "buildSql" method. What do you think about this?